README.md

```diff
@@ -36,7 +36,7 @@ on the docs website.
 Other useful links:
```

Comment:
This whole file has been re-organized and points to awesome-flame for tutorials, so the changes to this file could be removed.